src/Bundler.js

```diff
@@ -532,7 +538,7 @@ class Bundler extends EventEmitter {
       bundle = Bundle.createWithAsset(asset);
     } else if (dep && dep.dynamic) {
       // Create a new bundle for dynamic imports
-      bundle = bundle.createChildBundle(asset);
+      bundle = bundle.createChildBundle(asset, dep);
```

Comment:
Is this the magical line that will allow me to 

```jsx
import { debounce } from 'lodash';
```

...in my web workers?  We're very keen on being able to bundle external assets in our webworker scripts

Reply:
This PR should fix it